include/oneapi/tbb/detail/_template_helpers.h

```diff
@@ -184,6 +184,15 @@ class raii_guard {
 public:
     raii_guard( Func f ) : my_func(f), is_active(true) {}
 
+    raii_guard (raii_guard&& g) : my_func(g.my_func), is_active(g.is_active) {
+        g.is_active = false;
+    }
+    raii_guard& operator=(raii_guard&& g) {
```

Comment:
Do we really need this function ? 
IMHO both assignment operators for `raii_guard` should be banned. 


And if we do __realy__ need them - first question what if the object being assigned is already armed (i.e. has function assigned and `is_active == true`) ?

Reply:
I agree that the `raii_guard` object assignment seems strange. This member was removed from the interface.